web/src/main/java/org/springframework/security/web/authentication/rememberme/AbstractRememberMeServices.java

```diff
@@ -424,12 +419,7 @@ protected void setCookie(String[] tokens, int maxAge, HttpServletRequest request
 			cookie.setSecure(useSecureCookie);
 		}
 
-		if (setHttpOnlyMethod != null) {
```

Comment:
Will you please double-check the javadoc for the `setCookie` method here? I believe it still refers to Servlet 3.0 on line 394.

Reply:
@jzheaux yeah, you're right. thank you.
i missed the javadoc. i'm gonna fix it.